common/setups/rasr/nn_system.py

```diff
@@ -66,3 +66,34 @@ def __init__(
         )
 
         self.blas_lib = blas_lib or (gs.BLAS_LIB if hasattr(gs, "BLAS_LIB") else None)
+
+        self.native_ops = {}  # type: Dict[str, tk.Path]
+
+    def compile_native_op(self, op_name: str):
+        """
+        Compiles and stores a native op for RETURNN
+
+        :param op_name: op name, e.g. "NativeLstm2"
+        """
+        native_lstm_job = returnn.CompileNativeOpJob(
```

Comment:
```suggestion
        native_op_job = returnn.CompileNativeOpJob(
```
Same applies a few lines down.